src/components/ListItem.js

```diff
@@ -59,26 +67,57 @@ const ListItem = ({ item, onDelete, token }) => {
   }
 
   return (
-    <li className={className}>
-      <span className="label" aria-hidden="true">
-        {purchaseNext}
-      </span>
-      {item.name}
-      <span className="screen-reader-only">
-        Next purchase in {item.next_purchase} days.
-      </span>
+    <li>
       <button
-        className={isPurchased ? 'purchased' : 'not-purchased'}
+        className={`${
+          isPurchased ? 'purchased' : 'not-purchased'
+        } ${className}`}
         onClick={onHandle}
         disabled={isPurchased ? !null : null}
       >
-        Purchase
-        <span className="screen-reader-only">{item.name}.</span>
+        {isPurchased ? (
+          <img
+            className="egg"
+            src={eggHeartCracked}
+            title="Item is purchased"
+            alt="Item is purchased"
+          />
+        ) : (
+          <img
+            className="egg"
+            src={eggWhole}
+            title="Purchase item"
+            alt="Purchase item"
+          />
+        )}
+
+        <span className="screen-reader-only">Purchase {item.name}.</span>
       </button>
-      <button onClick={onDelete}>
-        <img className="trash" src={trash} alt="delete item" />
+
+      <span className="item-name">{item.name}</span>
+
+      <span className="screen-reader-only">
+        Next purchase in {item.next_purchase} days.
+      </span>
+
+      <button className="trash" onClick={onDelete}>
+        <img
+          className="trashIcon"
+          src={trash}
+          title="Delete item"
+          alt="Delete item"
+        />
       </button>
-      <Link to={`detail/${item.name_normalized}`}>View Details</Link>
+
+      <Link to={`detail/${item.name_normalized}`}>
+        <FontAwesomeIcon
+          icon={faAngleRight}
+          size="3x"
+          className="chevron"
+          title="View Details"
```

Comment:
```suggestion
          title={`View ${item.name} Details`}
```
If you want to have the item name...not a big deal if you don't.